src/backends/pyplot.jl

```diff
@@ -1202,13 +1225,26 @@ function _update_min_padding!(sp::Subplot{PyPlotBackend})
         end
     end
 
-    # optionally add the width of colorbar labels and colorbar to rightpad
-    if haskey(sp.attr, :cbar_ax)
-        bb = py_bbox(sp.attr[:cbar_handle]."ax"."get_yticklabels"())
-        sp.attr[:cbar_width] = width(bb) + (sp[:colorbar_title] == "" ? 0px : 30px)
-        rightpad = rightpad + sp.attr[:cbar_width]
+    if haskey(sp.attr, :cbar_ax) # Treat colorbar the same way
```

Comment:
I think we might as well treat colorbar the same (as in matplotlib it is too an ax)